lib/matplotlib/axes/_axes.py

```diff
@@ -5380,6 +5381,12 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
             which can be set by *filterrad*. Additionally, the antigrain image
             resize filter is controlled by the parameter *filternorm*.
 
+        interpolation_stage : {'data', 'rgba'}, default: 'data'
+            Supported values are 'data' and 'rgba'.  If 'data', interpolation
```

Comment:
The supported values are implied from the type line (also, re-wrap.)
```suggestion
        interpolation_stage : {'data', 'rgba'}, default: 'data'
            If 'data', interpolation
```